src/render/color.rs

```diff
@@ -1,8 +1,14 @@
 //! Colors used for drawing to a Cairo buffer
 
 use std::convert::From;
+use std::str::Chars;
+
+const WLC_BUG_INVERT_RED_BLUE: bool = true;
```

Comment:
I suggest removing this and leaving the comments. Once we switch away from WLC there's no need for this. Also it's unknown if this whole module will survive the trip there, though it probably will. It'll be pretty obvious when it doesn't work thankfully, and I think this boolean check just adds noise.

e.g I never see myself setting this to `false`. I'll just delete the old code.

Reply:
Fair point - I'll change it.